### DISCUSSION TITLE
Disable "SAT"

### DISCUSSION THREAD

**Discussion by antonov124** [magicbug/Cloudlog#1637] [Q&A]
Is it possible to switch off "Satellite"  or Sat in all Cloudlog Menus.?

**Comment by phl0** [comment]
> Is it possible to switch off "Satellite" or Sat in all Cloudlog Menus.?

On station_logbooks branch there is an implementation to disable certains bands and SAT. Have you tried that? Was added with: https://github.com/magicbug/Cloudlog/pull/1595. This can be used to exclude SAT from analytics and awards at least.

**Comment by antonov124** [comment]
unfortunately i can't do anything with this information, because i'm not so fit in linux or editing. how can i switch to this branch?

**Comment by magicbug** [comment]
As with most software "satellite" box is shown on QSO entry (HRD etc) however in v2 of cloudlog you'll be able to hide it from awards.

**Reply by phl0** [reply]
I am just extending this a little so that the SAT stuff on the other pages is hidden if the user de-selects the SAT band.

**Comment by phl0** [comment]
I have a patch under https://github.com/magicbug/Cloudlog/compare/station_logbooks...phl0:Cloudlog:hideSatStuff?expand=1. That would hide the SAT stuff if the band is disabled in band settings. What do you think @magicbug? Let me know if I should PR it.
